config/routes.rb

```diff
@@ -35,6 +35,8 @@ def patch(*)
     post "webhooks/aws" => "webhooks#aws"
     post "webhooks/mailgun" => "webhooks#mailgun"
     post "webhooks/mailjet" => "webhooks#mailjet"
+    post "webhooks/mailgun" => "webhooks#mailgun"
```

Comment:
Looks like you accidentally duplicated the mailgun line

Reply:
Oops, that was a merge conflict resolved wrong.